rust/olm-deployer/README.md

```diff
@@ -0,0 +1,23 @@
+# How to test
+
+Requirements:
+
+1. An OpenShift cluster.
+2. Checkout the branch `secret-olm-deployer` from the [operators](https://github.com/stackabletech/openshift-certified-operators/tree/secret-olm-deployer) repo.
+3. Clone the `stackable-utils` [repo](https://github.com/stackabletech/stackable-utils)
+
+Install the secret operator using OLM and the `olm-deployer`. From the `stackable-utils` repo, run:
+
+```bash
+$ ./olm/build-bundles.sh -c $HOME/repo/stackable/openshift-certified-operators -r 24.11.0 -o secret -d
```

Comment:
as we are now pushing to OCI, it is necessary to login first with:
`docker login oci.stackable.tech` and then supply username plus a CLI key obtained from User/Profile in the Harbor UI

Reply:
[Done](https://github.com/stackabletech/secret-operator/pull/546/commits/00520b49478923eace7ad5bca314f218cce31d2a)

Reply:
Thanks!